core/domain/question_services.py

```diff
@@ -465,6 +568,7 @@ def _save_question(committer_id, question, change_list, commit_message):
     question_model.language_code = question.language_code
     question_model.question_state_data_schema_version = (
         question.question_state_data_schema_version)
+    question_model.linked_skill_ids = set(question.linked_skill_ids)
```

Comment:
question_model.linked_skill_ids should be a list right? According to your backend tests above.

Reply:
The reason I do this is to remove duplicates. It gets stored as a list (and we do not need it to be ordered)

Reply:
Suggest doing list(set(...)) for clarity.

Reply:
Done